manifests/h/HagelTechnologiesLtd/DUMeter/8.01/HagelTechnologiesLtd.DUMeter.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HagelTechnologiesLtd.DUMeter
+PackageVersion: 8.01
+PackageLocale: en-US
+Publisher: Hagel Technologies Ltd.
+PackageName: Du Meter
+License: Copyright © 1997-2022 Hagel Technologies Ltd.

```

A copyright is not a license 

```suggestion
License: Proprietary
LicenseUrl: https://www.hageltech.com/dumeter/license
Copyright: Copyright © 1997-2022 Hagel Technologies Ltd.
CopyrightUrl: https://www.hageltech.com/dumeter/license
```